hal/riscv64/pmap.c

```diff
@@ -374,14 +385,9 @@ void _pmap_init(pmap_t *pmap, void **vstart, void **vend)
 			pmap_common.minAddr = a;
 	}
 
-/* TODO: add new syspage */
-#if 0
-	void *v;
-	u64 e = (u64)_end;
-
 	/* Initialize kernel page table - remove first 4 MB mapping */
 	pmap->pdir2 = pmap_common.pdir2;
-	pmap->satp = ((syspage->pdir2 >> 12) | (u64)0x8000000000000000);
+	pmap->satp = (((u64)pmap_common.pdir2 >> 12) | (u64)0x8000000000000000);
```

Comment:
I see there is some existing code that uses u64 instead of addr_t/ptr_t. It would be good to clean this up

Reply:
I have cleaned up the code and pushed it in the new commit.